Configuration/TCA/Overrides/sys_file_reference.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;
+
+$displayCondition = [
+    'AND' => [
+        'FIELD:tablenames:=:tx_html5mediakit_domain_model_media',
+        'FIELD:fieldname:=:tracks',
+    ],
+];
+
+$additionalColumns = [
+    'tx_html5mediakit_track_kind' => [
+        'label' => 'LLL:EXT:html5mediakit/Resources/Private/Language/locallang_db:tx_html5mediakit_track_kind',
```

Comment:
Please add the tablename (sys_file_reference) as a prefix for the translations keys, e.g. in this case:

```suggestion
        'label' => 'LLL:EXT:html5mediakit/Resources/Private/Language/locallang_db:sys_file_reference.tx_html5mediakit_track_kind',
```